# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGIntegrationUtils.cpp
# Commit: 98bea2812b17
# Full Hash: 98bea2812b17ee0294eb90b5e37de904412e659b
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-10-24 03:54:44
# Regressor Bug: 1765202
# File Overlap Count: 1
# Description:
#   Bug 1765202 part 1 - Have determineMaskUsage return its result r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190766
# ==============================================================================

diff -r 79f439bc112c -r 98bea2812b17 layout/svg/SVGIntegrationUtils.cpp
--- a/layout/svg/SVGIntegrationUtils.cpp	Mon Oct 23 20:00:05 2023 +0000
+++ b/layout/svg/SVGIntegrationUtils.cpp	Mon Oct 23 20:11:10 2023 +0000
@@ -642,8 +642,8 @@
                                     bool& aOutIsMaskComplete) {
   aOutIsMaskComplete = true;
 
-  SVGUtils::MaskUsage maskUsage;
-  SVGUtils::DetermineMaskUsage(aParams.frame, aParams.handleOpacity, maskUsage);
+  SVGUtils::MaskUsage maskUsage =
+      SVGUtils::DetermineMaskUsage(aParams.frame, aParams.handleOpacity);
   if (!maskUsage.shouldDoSomething()) {
     return false;
   }
@@ -767,8 +767,8 @@
     return;
   }
 
-  SVGUtils::MaskUsage maskUsage;
-  SVGUtils::DetermineMaskUsage(aParams.frame, aParams.handleOpacity, maskUsage);
+  SVGUtils::MaskUsage maskUsage =
+      SVGUtils::DetermineMaskUsage(aParams.frame, aParams.handleOpacity);
 
   if (maskUsage.opacity == 0.0f) {
     return;